app/graphql/types/entitlement/plan_entitlement_object.rb

```diff
@@ -7,20 +7,27 @@ class PlanEntitlementObject < Types::BaseObject
 
       field :code, String, null: false
       field :description, String, null: true
-      field :feature, Types::Entitlement::FeatureObject, null: false
       field :name, String, null: false
-      field :privileges, [PlanEntitlementPrivilegeObject], null: false, method: :values
+      field :privileges, [PlanEntitlementPrivilegeObject], null: false
+
+      # NOTE: For regular plans, the attribute are read on the object
+      #       For plan override, we're reading the *subscrition entitlements*
+      #       See `Types::Plans::Object.entitlements` note for more details.
 
       def code
-        object.feature.code
+        object.is_a?(Hash) ? object.fetch(:code) : object.feature.code
```

Comment:
this make me unconfortable

Reply:
We're changing the form design so I was able to remove this 🙌